docs/reference/mapping/dynamic/templates.asciidoc

```diff
@@ -118,10 +118,12 @@ to map `string` fields as either indexed fields or runtime fields.
 [[match-mapping-type]]
 ==== `match_mapping_type`
 
-The `match_mapping_type` is the data type detected by the JSON parser. Because
-JSON doesn't distinguish a `long` from an `integer` or a `double` from
-a `float`, it always chooses the wider data type such as `long` for integers
-and `double` for floating-point numbers.
```

Comment:
@javanna, given the discussions in #82965, I think that this statement is not true.

Reply:
++

Reply:
I wonder if we should still mention that JSON does not distinguish, and in short, any parsed floating point number is considered a double json data type, while any parsed integer number is considered a long. Up to you.

Reply:
Thanks @javanna -- I'll add that clarification.